README.md

```diff
@@ -0,0 +1,64 @@
+# Go SafeCast
+
+## Origin of this project
+
+In Go, integer type conversion can lead to unexpected behavior and errors if not handled carefully.
+When converting between integer types, issues can happen when converting between signed and unsigned integers, or when converting to a smaller integer type. The code will panic.
```

Comment:
When does it ever panic? can you add a play.go.dev link of code that does and/or an example?

It seems a bit repetitive "unexpected behavior and errors" isn't it the same?

Reply:
You are right, it doesn't panic. I updated the README